browser/resources/settings/brave_sync_page/brave_sync_configure.js

```diff
@@ -82,11 +78,20 @@ Polymer({
     return displayDate.toDateString()
   },
 
-  onViewSyncCode_: function() {
+  getSyncCode_: async function() {
+    if (!!this.syncCode)
```

Comment:
might be useful to add a comment like "only set `syncCode` if it's not already set

Reply:
per our call - this needs to be cleared when you leave the chain

Reply:
fixed